src/components/admin/agency/logo-upload.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class LogoUpload extends React.Component {
```

Comment:
I'm not sure where you get this code, but we don't use state, out of the constructor. Please use state initialization inside the constructor function.